cpp/README.md

```diff
@@ -0,0 +1,6 @@
+# C++ implementation of the MCAP (Message Capture) library
```

Comment:
we've stopped using the phrase "message capture" so you can just remove that from here and conanfile.py